packages/meditrak-app/app/entityMenu/helpers.js

```diff
@@ -34,7 +34,7 @@ export const getEntityAttributeChecker = (state, questionId) => {
   }
 
   return entity =>
-    Object.entries(JSON.parse(attributes)).every(([key, config]) => {
```

Comment:
Because this is running after `Realm` converts it to a value object it is already parsed into a JSON object